newIDE/app/src/AssetStore/InstallAsset.spec.js

```diff
@@ -6,7 +6,6 @@ import {
   filterMissingBehaviors,
   downloadExtensions,
   installAsset,
-  getRequiredExtensionsForEventsFromAsset,
```

Comment:
I'm curious what problem this capability was giving you?

Reply:
The asset store wants an EventList to add events on the object.
I could have done a new dialog to create an object without the asset store tab, but I wasn't sure if I should as the asset store could be interesting to have placeholders while designing an extension. I was more convinced that adding events directly at the end of the scene events was not something to do.